contracts/interfaces/modules/royalty/policies/IRoyaltyPolicyLAP.sol

```diff
@@ -19,6 +19,20 @@ interface IRoyaltyPolicyLAP is IRoyaltyPolicy {
         uint32[] targetRoyaltyAmount
     );
 
+    /// @notice The state data of the LAP royalty policy
+    /// @param isUnlinkableToParents Indicates if the ipId is unlinkable to new parents
+    /// @param ipRoyaltyVault The ip royalty vault address
+    /// @param royaltyStack The royalty stack of a given ipId is the sum of the royalties to be paid to each ancestors
+    /// @param ancestorsAddresses The ancestors addresses array
+    /// @param ancestorsRoyalties The ancestors royalties array
```

Comment:
Can you expand on this comment, ie. what "royalty" of ancestors?

Reply:
https://github.com/storyprotocol/protocol-core-v1/pull/35/commits/213e2b7ad4314abc59b273e79d364c384a80499f